Makefile

```diff
@@ -47,7 +47,7 @@ manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and Cust
 		output:rbac:artifacts:config=manifests/v2/base/rbac \
 		output:webhook:artifacts:config=manifests/v2/base/webhook
 
-generate: controller-gen manifests ## Generate apidoc, sdk and code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
+generate: go-mod-download controller-gen manifests ## Generate apidoc, sdk and code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
```

Comment:
Btw, I think you can remove `controller-gen` task from the `generate`, since `manifests` runs `controller-gen`: https://github.com/kubeflow/training-operator/blob/952f8452843d9fedf3015938e410ed2bd6121623/Makefile#L39

Reply:
Good point, updated.